### PR TITLE
Unlock symfony 5 in develop branch

### PR DESCRIPTION
**Pull request by rabauss**
For Contao 4.13 we need support for symfony 5 in the develop branch!

### CODE REVIEW

**Review by fritzmg — CHANGES_REQUESTED** [review]
This will break any existing listener - you still need to provide the event's name as the second parameter (see also the discussion in  https://github.com/contao/contao/pull/4321). Also you need to drop the Symfony `^3.4` requirement as this will not work in Symfony 3.

File: composer.json [hunk 1]
```diff
@@ -12,9 +12,17 @@
     ],
     "require":{
         "php":"^7.1 || ^8.0",
```
Comment (rabauss) [inline composer.json:14]: Maybe we should even bump up the php version

Comment (rabauss) [reply composer.json:14]:
```suggestion
        "php":"^7.4 || ^8.0",
```

**Comment by aschempp** [comment]
Thank you @rabauss

**Pull request merged**